app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -1,4 +1,4 @@
-child :content_facet => :content do
+child :content_facet => :content_facet_attributes do
```

Comment:
If it's content_facet here why does it get mapped to have `attributes` attached at the end? You can point me at the code for this I just don't recall ever seeing it.